test/wrapper.auto.tfvars

```diff
@@ -40,9 +40,10 @@ maintenance_policy_config = [
 ]
 enable_cost_allocation_feature = true
 enable_l4_ilb_subsetting       = true
+deletion_protection            = false
 
 release_channel    = "RAPID"
-kubernetes_version = "1.30.1-gke.1156000"
+kubernetes_version = "1.30.2-gke.1587003"
```

Comment:
This seems fragile.  Maybe better to no set explicitly and either let the module work out itself, or query from the test and set the input there.

Reply:
@marty This can be part of improvement, In the test we are directly calling the [child module](https://github.com/honestbank/terraform-gcp-gke/blob/6b5164180b415cd036e4e348e119a4720c69df9a/test/gke_test.go#L93) and passing all the required variables using auto tfvars,.

For this we need the update the test logic or put the conditional statement in the child module.Have created Backlog ticket for this: 

https://linear.app/honestbank/issue/DEVOP-4864/update-the-gcp-gke-module-terratest-to-dynamically-fetch-gke-version